reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -319,21 +319,29 @@ A cast can also be performed when a variable is assigned to using [cast notation
 
 #### Comma operator `,`
 
-As a binary operator, the comma creates an array. As a unary operator, the
-comma creates an array with one member. Place the comma before the member.
+As a binary operator, the comma creates an array or appends to the array being
+created. In expression mode, as a unary operator, the comma creates an array
+with just one member. Place the comma before the member.
 
 ```powershell
 $myArray = 1,2,3
 $SingleArray = ,1
+Write-Output (,1)
 ```
 
 #### Dot sourcing operator `.`
 
+<!--
+01234567890123456789012345678901234567890123456789012345678901234567890123456789
+-->
 Runs a script in the current scope so that any functions, aliases, and
-variables that the script creates are added to the current scope.
+variables that the script creates are added to the current scope, overriding
+existing ones. Parameters declared by the sourced script become variables too.
+Parameters for which no value has been given become variables with no value.
+However, the automatic variable `$args` is preserved.
 
 ```powershell
-. c:\scripts\sample.ps1
+. c:\scripts\sample.ps1 1 2 -Also:3
```

Comment:
```suggestion
. c:\scripts\sample.ps1 1 2 -Also 3
```

Reply:
Why exactly?  `-Also 3` is ambiguous, **Also** might be a switch.

Reply:
I think this example is unnecessarily complex. `1` and `2` could be positional params  but without an actuall script to match to, the user won't know and really not have a reference. They aren't needed to highlight the behavior you are talking about. I think having something staight forward is a better fit here where it's extremely shallow in informational depth. 

`. c:\scripts\sample.ps1 -TestParam 'TestValue'`

It could be read as a switch but in reality, that's not the standard and there is no specific reference to a switch you are trying to highlight. We should be trying to highlight that the parameter is given a value and the user can understand that `-TestParam` will be available as `$TestParam`.  I don't see how `1` or `2` help highlight the behavior above?

Reply:
I think we need to show the contents of a simple script and the results of dot-sourcing.

sample.ps1
```
[cmdletbinding()]
param([int]$timeout)

"The timeout value is $timeout."
```

Execution example:
```
PS> .\sample.ps1 -timeout 4
The timeout value is 4.
PS> $timeout
PS> . .\sample.ps1 -timeout 4
PS> $timeout
4
```